contrib/experimental/great_expectations_experimental/expectations/expect_queried_column_to_be_unique_with_condition.py

```diff
@@ -70,9 +70,8 @@ def _validate(
         runtime_configuration: dict = None,
         execution_engine: ExecutionEngine = None,
     ) -> Union[ExpectationValidationResult, dict]:
-
         query_result = metrics.get("query.template_values")
```

Comment:
same question

Reply:
@Shinnnyshinshin Thank you -- just trying to use it only when absolutely necessary.